README.md

```diff
@@ -0,0 +1,13 @@
+# Gmp-detector
+
+This repository's aim to contain a reusable script, to detect all GMP function usages in different repositories.
+
+In order to do that we create a simple script, which lists all the functions in a specified folder.
```

Comment:
The content is pretty good, however, there are some smaller grammatical issues. Can you check it with grammarly, maybe?

Reply:
Sure!